tests/route/test_default_route.py

```diff
@@ -112,13 +115,13 @@ def verify_default_route_in_app_db(duthost, tbinfo, af, uplink_ns):
 
 
 
-def test_default_route_set_src(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_asic_index, tbinfo):
+def test_default_route_set_src(duthosts, tbinfo):
     """
     check if ipv4 and ipv6 default src address match Loopback0 address
 
     """
-    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
-    asichost = duthost.asic_instance(enum_asic_index)
+    duthost = find_duthost_on_role(duthosts, get_upstream_neigh_type(tbinfo['topo']['name']) , tbinfo)
```

Comment:
where is `find_duthost_on_role`?

Reply:
it has been merged as part of everflow PR